hutch_python/tests/test_plan_wrappers.py

```diff
@@ -0,0 +1,165 @@
+import logging
+from typing import Any, Callable
+
+import pytest
+from bluesky import RunEngine
+from bluesky.plan_stubs import null, pause
+from bluesky.plans import scan
+from bluesky.utils import RunEngineInterrupted
+from ophyd.sim import motor
+
+from hutch_python.plan_wrappers import (ImproperRunWrapperUse, PlanWrapper,
+                                        RunEngineWrapper,
+                                        initialize_wrapper_namespaces,
+                                        register_plan, registry)
+from hutch_python.utils import HelpfulNamespace
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def RE() -> RunEngine:
+    return RunEngine(call_returns_result=True)
+
+
+@pytest.fixture(scope='function')
+def run_scan(RE: RunEngine) -> Callable:
+    return RunEngineWrapper(scan, RE)
+
+
+def do_standard_check(run_scan: Callable):
+    motor.set(5)
+    result = run_scan([], motor, 0, 10, 11)
+    assert result.exit_status == 'success'
+    assert motor.position == 10
```

Comment:
This was an interesting assertion / test setup to me, since it assumes you're running a very specific plan.   `do_standard_check` seemed like something that might be plan-agnostic, but it's really not.

Reply:
You're right in that I picked a bad name for this utility function. If this was in the main code area or if I wasn't emotionally done with this PR I would absolutely go back and rename the function. But since it isn't in the main code area I'll just make a mental note to extend clear function names to test suites in the future.